os/kernel/init/os_bringup.c

```diff
@@ -115,6 +115,15 @@
 #endif
 #endif
 
+#if defined(CONFIG_EXAMPLES_ELF)
+/* Loadable APPS configuration */
+#define NUM_APPS	2
```

Comment:
Don't include board specific code here.

Reply:
It is not a board specific code.  
Anyway it is a temporary  location to launch the  loadable apps from the kernel for DA Micom-WiFi integration use case.

Reply:
"Anyway it is a temporary location to launch the loadable apps from the kernel for DA Micom-WiFi integration use case" - can't find more convincing reason to push into board specific. 

There are **board specific hooks for these.** This code don't deserve to be  here and there is NO reason for it.

Reply:
Ok. if the requirement is to start loadable apps execution from kernel, do you have any better place to put this code?

Reply:
oh, you just mean  `#define NUM_APPS	2` ?     :)

This will be changed, when we add proper configuration for app binary separation.  This one is just added temporarily.  Otherwise if possible I will try to add some configuration under EXAMPLE_ELF..